### PR TITLE
Fixed DOC-654 by documenting innodb_ft_ignore_stopwords (5.7)

### PR DESCRIPTION
**Pull request by hrvojem**
fixed dep8 issues in older RNs and a build warning

### CODE REVIEW

File: doc/source/flexibility/innodb_fts_improvements.rst [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+InnoDB Full-Text Search improvements
+====================================
+
+By default n-gram indexes check the `stopwords list
+<https://dev.mysql.com/doc/refman/5.7/en/fulltext-stopwords.html>`_,
+to see if any indexed elements *contain* one of the words on that list.
+
+Using this list for n-gram indexes isn't always suitable, as an example, any
+item that contains ``a`` or ``i`` will be ignored. Another word that can't be
+searched is ``east``, this one will find no matches because ``ea`` is on the
```
Comment (percona-ysorokin) [inline doc/source/flexibility/innodb_fts_improvements.rst]: Although this is exactly what is stated in the blueprint document `ea` is not a default stopword.
I would suggest to change this to just `a`.

File: doc/source/flexibility/innodb_fts_improvements.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+.. variable:: innodb_ft_ignore_stopwords
+
+  :cli: Yes
+  :conf: Yes
+  :scope: Session, Global
+  :dyn: Yes
+  :vartype: Boolean
+  :default: ``OFF``
+
+When enabled, this variable will instruct |InnoDB| Full Text Search n-gram
```
Comment (percona-ysorokin) [inline doc/source/flexibility/innodb_fts_improvements.rst]: Remove `n-gram`. This is true for other parsers as well.

**Comment by hrvojem** [comment]
Fixed

File: doc/source/flexibility/innodb_fts_improvements.rst [hunk 3]
```diff
@@ -0,0 +1,48 @@
+.. _innodb_fts_improvements:
+
+====================================
+InnoDB Full-Text Search improvements
+====================================
+
+By default n-gram indexes check the `stopwords list
```
Comment (laurynas-biveinis) [inline doc/source/flexibility/innodb_fts_improvements.rst]: Make a new subsection for stopword ignoring

Comment (laurynas-biveinis) [inline doc/source/flexibility/innodb_fts_improvements.rst]: All FTS indexes

File: doc/source/flexibility/innodb_fts_improvements.rst [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  :dyn: Yes
+  :vartype: Boolean
+  :default: ``OFF``
+
+When enabled, this variable will instruct |InnoDB| Full Text Search
+parser to ignore the stopword list when building/updating an FTS index.
+
+.. note::
+
+   Although this variable is introduced to resolve n-gram issues, it affects
```
Comment (laurynas-biveinis) [inline doc/source/flexibility/innodb_fts_improvements.rst]: BP description is usually not suitable for copy-pasting into docs directly. This note should be merged to the description above

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]

**Comment by hrvojem** [comment]
Fixed

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**